modules/ezcontent_node/modules/ezcontent_smart_article/css/ezcontent_smart_article.css

```diff
@@ -38,3 +38,7 @@
   background-color: var(--grey-color);
   opacity: 0.4;
 }
+
+.hide {
```

Comment:
This styling exists in core `core/modules/system/css/components/hidden.module.css`
Can we reuse ?

Reply:
`.button` class, overrides `.hidden` class styling, hence this class is required.